src/widgetastic/widget/base.py

```diff
@@ -1015,6 +1016,19 @@ def after_fill(self, was_change):
         """
         pass
 
+    def child_widget_accessed(self, widget):
+        """Called when a child widget of this widget gets accessed.
```

Comment:
`child widget of this view` <- I think its good to be more explicit here, even though View inherits Widget.